src/script/miniscript.h

```diff
@@ -1152,12 +1152,12 @@ struct Node {
      * the script push. */
     std::optional<uint32_t> GetStackSize() const {
         if (!ss.sat.valid) return {};
-        return ss.sat.value + 1;
+        return ss.sat.value;
```

Comment:
In ecdee0c3f802c6308c358a5ee501d2e031c8dcf2 "miniscript: make GetStackSize independent of P2WSH context"

The comment above this function is outdated.

Reply:
Updated the comment.